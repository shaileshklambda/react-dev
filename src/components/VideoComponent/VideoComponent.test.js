import React from "react";
import renderer from "react-test-renderer";
import Videocomponent from ".";

it("Videocomponent: default", () => {
  const component = renderer.create(<Videocomponent />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
