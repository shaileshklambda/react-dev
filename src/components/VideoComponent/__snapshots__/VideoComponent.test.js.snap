// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Videocomponent: default 1`] = `
<div>
  <div
    data-vjs-player={true}
  >
    <video
      className="video-js vjs-big-play-centered"
      poster=""
    />
  </div>
</div>
`;
